dts/arm/nordic/nrf54l15_cpuapp.dtsi

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <arm/armv8-m.dtsi>
+#include <nordic/nrf_common.dtsi>
+#include <nordic/nrf54l_common.dtsi>
+
+/ {
+	chosen {
+		zephyr,ieee802154 = &ieee802154;
+		zephyr,entropy = &cracen;
+	};
```

Comment:
choices should be avoided in soc/ dts

Reply:
`ieee802154` moved to `boards/arm/nrf54l15pdk_nrf54l15/nrf54l15pdk_nrf54l15_cpuapp.dts`, entropy removed.

Reply:
also comment from above mentioned that cracen is not available in the upstream